Makefile

```diff
@@ -141,7 +141,7 @@ e2e:
 
 .PHONY: unit-test
 unit-test:
-	go test $(shell go list ./... | grep -v /test/e2e) -v
+	go test -count=1 $(shell go list ./... | grep -v /test/e2e) -v
```

Comment:
What is the usage of count here ?

Reply:
It's to ensure the test runs at least once instead of showing the cached result.